katello/katello/db_config.rb

```diff
@@ -0,0 +1,66 @@
+require 'yaml'
```

Comment:
Since this so heavily ties into katello-installer, should this be part of the katello-installer repository?